assets/less/content/general.less

```diff
@@ -165,6 +165,10 @@ img {
   max-width: 100%;
 }
 
+img[src*="#gh-dark-mode-only"] {
```

Comment:
Alternatively we could do:

```
body:not(.night-mode) img[src*="#gh-dark-mode-only"] {
  display: none;
}
```

This way we don't need to explicitly do `display: inline` to revert this, which is good because the image may potentially have different `display`.

Maybe it would make sense to have both of these together as `body:not(.night-mode) img` and `body.night-mode img` in a single file?

Reply:
> Maybe it would make sense to have both of these together as body:not(.night-mode) img and body.night-mode img in a single file?

I am following the existing pattern, worth noting that this is only inside the `.content-inner`